CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ x.y.z Release notes (yyyy-MM-dd)
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-swift/issues/????), since v?.?.?)
 * `@AutoOpen` and `@AsyncOpen` failed to use the `initialSubscriptions` set in
   the configuration passed to them ([PR #8572](https://github.com/realm/realm-swift/pull/8572), since v10.50.0).
+* Accessing `App.currentUser` from within a notification produced by `App.switchToUser()` 
+  (which includes notifications for a newly logged in user) would deadlock. 
+  ([Core #7670](https://github.com/realm/realm-core/issues/7670), since v14.6.0).
+* Inserting the same link to the same key in a dictionary more than once would incorrectly
+  create multiple backlinks to the object. This did not appear to cause any crashes later, but would 
+  have affecting explicit backlink count queries (eg: `$0.links.count`) and possibly notifications. 
```

Comment:
We either don't support this query at all or it has a different syntax.

Reply:
I think this is referring to queries on backlinks for example `$0.linkingObjects.count == 3` where 
```
class TestObject: Object {
    @Persisted var objectCol: TestObject?

    @Persisted(originProperty: "objectCol")
    var linkingObjects: LinkingObjects< TestObject >
}
```
And the return count will be wrong because it will create multiple backlinks instead of just one. 
Correct me if I'm wrong @tgoyne